storage/shard.go

```diff
@@ -1714,9 +1735,20 @@ func (s *dbShard) Flush(
 			return false
 		}
 
+		// Debug
```

Comment:
could you remove the `// debug` indicators if they're going to be kept around.

Reply:
done